core/frontend/src/IModelConnection.ts

```diff
@@ -256,6 +257,16 @@ export abstract class IModelConnection extends IModel {
     return new ECSqlReader(executor, ecsql, params, config);
   }
 
+  /**
+   * queries every passed categoryId for its parentId, id, and appearance.
+   * @param compressedCategoryIds compressed category Ids
+   * @returns array of SubCategoryResultRow
+   *
+   */
+  public async queryCategoryIds(compressedCategoryIds: CompressedId64Set): Promise<SubCategoryResultRow[]> {
```

Comment:
`queryCategoryIds` does not describe what this function does. I'm not sure this and SubCategoryResultRow should be `@public`.

Reply:
What would be a better name?

queryCategoryIdsForSubCategories
getSubCategories
querySubCategories

Not sure...

Reply:
i opted for querySubCategories, and also changed the comment for the function. Does this describe it better?